__main__.py

```diff
@@ -33,5 +33,32 @@
 
 from libiocage.cli import cli
 
+
+def main_safe():
+  try:
+    main()
+  except BaseException as e:
+    return e
+
+
+def main():
+  cli(prog_name="iocage")
+
+
 if __name__ == "__main__":
-    cli(prog_name="iocage")
+
+  debug = False
+
+  try:
+    coverdir = os.environ["IOCAGE_TRACE"]
```

Comment:
This should be `coverdir = os.environ.get("IOCAGE_TRACE", DEFAULTVALUE)`, otherwise coverdir will be `None` and may cause issues.